dali/ft/filecopy.cpp

```diff
@@ -3259,18 +3261,20 @@ void FileSprayer::updateTargetProperties()
                 {
                     FilePartInfo & curSource = sources.item(idx);
                     RemoteFilename &remoteFile = curSource.filename;
-                    splitAndStoreFileInfo(newRecord, remoteFile, idx, false);
+                    splitAndCollectFileInfo(newRecord, remoteFile, fileNames, idx, false);
                 }
             }
+            newRecord->setProp("@name", fileNames.str());
```

Comment:
As mentioned in the JIRA, using an attribute when value is huge is problematic, speed/space/etc.
I assume you've kept it as an attribute for backward compatibility?

I doubt it's a big concern, changing it now would mean _if_ older build Esp's/dfuplus's tried to look at a newer build that had populated files with the new format they would not see the name correctly. (worth double checking that it doesn't cause any unexpected behaviour though - e.g. crash).

Reply:
This Origin (History) related part merged into 6.2 nearly 1.5 years ago. I tested the file metainfo query with large amount of filenames in Daliadmin and DFUPlus and I can't find any problem. So I would keep it.

Reply:
As I discussed in JIRA, - it's not just performance (speed)
It's also that it takes up a huge amount of space in Dali - and will not be compressed and will not transit between server and client compressed, so will impact network too.

Reply:
OK